airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java

```diff
@@ -69,28 +71,24 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ConfigDumpImport {
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class ConfigDumpImporter {
```

Comment:
Refactored this class so the constructor takes in the dbs so that it can be dependency injected, but then `import` can be called many times with data specific to the import.